### DISCUSSION TITLE
Idea for the Documentation - Live update

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [skeletonlabs/skeleton#904] [Ideas]
I believe this is something that "should" be possible.

The idea would be to have the components (For example this button on the button page) to update as we scroll through the list and not only once we actually select one option.

https://i.imgur.com/eQFhTgK.png

![](https://i.imgur.com/eQFhTgK.png)

**Comment by MrAmericanMike** [comment]
Let me add. I just noticed that after selecting one, if we use the arrows up and down, that causes a live update. But not while the menu (dropdown) is open. 

So I guess the functionality is there, it's probably some limitation for it to happen the way I described? Not sure...

**Reply by endigo9740** [reply]
These are native select inputs Mike, they work however native inputs work. I'm not sure this is something we need or want for the site. But thanks for suggestion.
